<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_follow_m</class>
 <widget class="QMainWindow" name="add_follow_m">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#234E70;
color: #FBF8BE;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="sumbit_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>420</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #FBF8BE;

   color:#1E2761;
}</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="anoc_edit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
	background-color: rgb(238, 238, 238);
color: rgb(0,0,0);
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Anot.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>135</width>
      <height>56</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="weight_label">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Weight [kg]</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="weight_edit">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	background-color: rgb(238, 238, 238);
color: rgb(0,0,0);
}
</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>21</y>
      <width>551</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="title_label">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Add New Folow-Up Meeting For:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="name_lable">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="cancel_button_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>420</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #FBF8BE;

   color:#1E2761;
}</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="add_m_calendarWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>312</width>
      <height>183</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>9</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(199, 199, 199);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="locale">
     <locale language="Spanish" country="Spain"/>
    </property>
   </widget>
   <widget class="QLabel" name="date_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="deat_label">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>62</y>
      <width>109</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Meeting Date</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="today_CB">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>78</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Today?</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
